core/src/utils/test/ready.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { componentOnReady } from '../helpers';
+
+describe('componentOnReady()', () => {
+  it('should correctly call callback for a custom element', (done) => {
+    customElements.define('hello-world', class extends HTMLElement {
+      constructor() {
+        super();
+      }
+    });
+
+    const component = document.createElement('hello-world');
+    componentOnReady(component, (el) => {
+      expect(el).toEqual(component);
```

Comment:
I would use `toBe` instead of `toEqual` here (and below) ?

Reply:
Ah yeah I always forget about that distinction